athena-federation-integ-test/src/main/java/com/amazonaws/athena/connector/integ/CloudFormationClient.java

```diff
@@ -140,9 +145,9 @@ protected void deleteStack()
         logger.info("------------------------------------------------------");
 
         try {
-            AmazonCloudFormation cloudFormationClient = AmazonCloudFormationClientBuilder.defaultClient();
             DeleteStackRequest request = new DeleteStackRequest().withStackName(stackName);
             cloudFormationClient.deleteStack(request);
+            cloudFormationClient.shutdown();
```

Comment:
shouldnt this be done in a finally block?

Reply:
Moved to finally block.